### PR TITLE
Community tutorials

### PR DESCRIPTION
**Pull request by ljgarcia**
This PR corresponds to an action assigned in the 2022.01.24 community call. 
It includes the IDP tutorial from PR https://github.com/BioSchemas/bioschemas.github.io/pull/505 and the Plant tutorial from PR https://github.com/BioSchemas/bioschemas.github.io/pull/503 using the new layouts and navigation

### CODE REVIEW

**Comment by ljgarcia** [comment]
@ivanmicetic please see that I have correctly included the IDP tutorial, thanks

**Comment by ljgarcia** [comment]
@DanFaria please see that I have correctly included the Plant tutorial. Tables do not render nicely, if you can improve them that would be great, thanks

File: _includes/community-tutorial-navigation.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<a href="{{ previousTutorial | relative_url}}">{{page.previousTutorial.title}}</a>
+{%- endif %}
+| <a href="/tutorials/community_tutorials">Community tutorials main page</a> |
+{%- if page.nextTutorial %}
+Next tutorial:
+{%- assign nextTutorial = '/tutorials/' | append: page.nextTutorial.link %}
+<a href="{{ nextTutorial | relative_url}}">{{ page.nextTutorial.title }}</a>
+<i class="fas fa-angle-right fa-lg ms-2"></i>
+{%- endif %}
+</div>
```
Comment (bedroesb) [inline _includes/community-tutorial-navigation.html:15]: this looks like a lot of duplication, maybe we can implement the option for a landing page also in the main one?

File: _layouts/community_tutorial.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      {% if page.bioschemas.dateCreated %}
+      <p><strong>Created:</strong> {{ page.bioschemas.dateCreated | date_to_long_string }}</p>
+      {% endif %}
+
+      {% if page.bioschemas.dateModified %}
+      <p><strong>Last Modified:</strong> {{ page.bioschemas.dateModified | date_to_long_string}}</p>
+      {% endif %}
+    </div>
+  </div>
+  {% include community-tutorial-navigation.html %}
```
Comment (bedroesb) [inline _layouts/community_tutorial.html:79]: Same here. Just my opinion!

**Review by bedroesb — COMMENTED** [review]
Maybe it is useful to first merge my latest PR, I can than help in bringing a landingpage option to the tutorial navigation

**Comment by bedroesb** [comment]
Just a small opinion I also want to share: I think it is always nice that Pull requests get merged/suggested for edits, and not just get closed and the changes integrated by another person. This for attribution on GitHub for the peoples work :)

**Comment by ljgarcia** [comment]
> Just a small opinion I also want to share: I think it is always nice that Pull requests get merged/suggested for edits, and not just get closed and the changes integrated by another person. This for attribution on GitHub for the peoples work :)

Sorry about that but the plant page would not render as it did not have any layout and the IDP would not be visible because it was under a child folder. I thought it was easier for IPD and Plant communities to see their tutorials already integrated into the new space for community tutorials so they can get a better idea on how it looks and so. My mistake, next time I will do my changes and will ping related PRs for them to directly handle them. Sorry about it. 

I managed to include the IDP PR into my PR via a merge commit but the Plant tutorial was outside the Bioschemas space so I did not know how to do the merge in that case.

**Comment by DanFaria** [comment]
@ljgarcia thanks! all seems fine on my end, including the tables in the file view I can see; I have no idea how they display on the website, but I can try to work on them (if needed) once I can visualize them there

**Comment by ivanmicetic** [comment]
It's ok with me too. I added some other commits to the old PR that are not included in this one (kramdown line numbering instead of manual JSON edits) but it does not matter, I will add them after this goes into production

File: _includes/community-tutorial-navigation.html (hunk 1 above)
Comment (ivanmicetic) [reply _includes/community-tutorial-navigation.html:15]: I don't quite understand what you're aiming at but overall community tutorials look good. Possibly one of the missing things is a link to community tutorials directly from main menu (Getting started)

**Review by ivanmicetic — APPROVED** [review]
Looks good.

**Review by AlasdairGray — CHANGES_REQUESTED** [review]
Some minor changes:
Overview page
- [x] Add links to the home page of each of the communities

@bedroesb
- [x] Add IDP and Plant tutorials to menu bar

IDP Tutorial: _could benefit from some rephrasing to improve english_
@ivanmicetic 
- [ ] What is the benefit to the IDP providers in embedding markup? Can we state that the data can be aggregated into the community resource IDPcentral?
- [ ] Can we add in the need to link to Scholarly Articles and link back to the dataset?
- [ ] Link to some real world example deployments

Plant Tutorial:
@DanFaria 
- [ ] Introduction, can we state that the data will be harvested into FAIDARE?
- [ ] Rather than Sample, can we use BioSample?
- [ ] Can you add in some examples of markup for a plant experiment?
- [ ] Can you add links to real world example deployments

File: _includes/community-tutorial-navigation.html (hunk 1 above)
Comment (ljgarcia) [reply _includes/community-tutorial-navigation.html]: @bedroesb if you see something is still missing, please go ahead and make the modifications

File: _layouts/community_tutorial.html (hunk 2 above)
Comment (ljgarcia) [reply _layouts/community_tutorial.html]: @bedroesb if you see something is still missing, please go ahead and make the modifications

**Comment by ljgarcia** [comment]
@ivanmicetic @DanFaria what would be the link to your communities?

File: _layouts/community_tutorial.html (hunk 2 above)
Comment (bedroesb) [reply _layouts/community_tutorial.html]: @ljgarcia I already made my modifications, thanks :)

**Comment by ivanmicetic** [comment]
> @ivanmicetic @DanFaria what would be the link to your communities?
you mean external link to community webpages? I would use the ones at ELIXIR: https://elixir-europe.org/communities/intrinsically-disordered-proteins and https://elixir-europe.org/communities/plant-sciences

**Comment by ljgarcia** [comment]
@ivanmicetic @DanFaria please see @AlasdairGray comments and implement the asked changes, if you need any help doing so, please ping me here. Thanks

**Comment by AlasdairGray** [comment]
> @ivanmicetic @DanFaria please see @AlasdairGray comments and implement the asked changes, if you need any help doing so, please ping me here. Thanks

@ljgarcia would it make sense to move my comments into separate issues that could then be applied later so that this PR can get merged in?

**Comment by ivanmicetic** [comment]
@ljgarcia @AlasdairGray I've rephrased the tutorial and added requested changes.

**Comment by ljgarcia** [comment]
> > @ivanmicetic @DanFaria please see @AlasdairGray comments and implement the asked changes, if you need any help doing so, please ping me here. Thanks
> 
> @ljgarcia would it make sense to move my comments into separate issues that could then be applied later so that this PR can get merged in?

@AlasdairGray yes, I think this PR is ready to go except for the improvements to the Plant tutorial. I suggest accept as it is and create new issue for @DanFaria

**Comment by ljgarcia** [comment]
@bedroesb I think you made changed to the layouts and templates for tutorials. Could you please have a look? Community tutorials are rendered with no layout. Thanks

**Comment by bedroesb** [comment]
@ljgarcia there was a mistake in the frontmatter, it should be fixed now

**Comment by bedroesb** [comment]
Screenshot from 2022-03-04 11-46-38

@AlasdairGray maybe we want to align the RD one with the other sentences?

**Comment by AlasdairGray** [comment]
> Screenshot from 2022-03-04 11-46-38
> 
> @AlasdairGray maybe we want to align the RD one with the other sentences?

Sure, feel free to make the changes

**Review by bedroesb — APPROVED** [review]

**Review by ljgarcia — COMMENTED** [review]
@bedroesb The link "[Return to community tutorials overview](/tutorials/community_tutorials.html)" on the community tutorials is broken. Please have a look. Thanks.

**Comment by ljgarcia** [comment]
@AlasdairGray once the link "Return to community tutorials overview" gets fixed, the PR would be ready to go. Please wait for this fix and re-review the PR. Thanks

**Comment by bedroesb** [comment]
@ljgarcia good catch! I changed ./community_tutorials.html into ./community/ because it made more sense from a URL standpoint. ./community/ has the index for community tutorials, ./community/tutorial_example the community tutorial itself.

**Comment by AlasdairGray** [comment]
@bedroesb the styling still does not seem to be coming through for the community tutorial pages

Screenshot 2022-03-10 at 15 12 01

**Comment by bedroesb** [comment]
Hello Alasdair, with me and on my fork it all looks fine? did you clear your cash and build from scratch?
Screenshot from 2022-03-10 16-39-11

[MY FORK](https://bedroesb.github.io/bioschemas.github.io/tutorials/community/idp)

**Comment by AlasdairGray** [comment]
It was a cache issue. Thanks

**Review by AlasdairGray — APPROVED** [review]

**Pull request merged**